lang/cs/build.props

```diff
@@ -70,7 +70,7 @@ under the License.
     <AvroVersion>1.5.6</AvroVersion>
     <NewtonsoftJsonVersion>8.0.3</NewtonsoftJsonVersion>
     <ProtobufVersion>2.0.0.668</ProtobufVersion>
-    <RxVersion>2.2.5</RxVersion>
+    <SystemReactiveVersion>3.1.1</SystemReactiveVersion>
```

Comment:
You might need to change the one used in build.DotNet.prop  as well for consistence.